eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/Relayer.scala

```diff
@@ -130,6 +130,7 @@ object Relayer extends Logging {
   }
 
   case class RelayForward(add: UpdateAddHtlc)
+
```

Comment:
Don't add an empty line here

Reply:
fixed in [ce0b37e](https://github.com/ACINQ/eclair/pull/2201/commits/ce0b37e6d83dbd9e0320cc46ba312f33331bb4b0)